src/Eseye.php

```diff
@@ -436,6 +398,79 @@ private function mapDataToUri(string $uri, array $data): string
         return $uri;
     }
 
+    /**
+     * Determine if call can be cached.
+     *
+     * @param  string  $method
+     * @return bool
+     */
+    private function isCachable(string $method): bool
+    {
+        return in_array(strtolower($method), $this->cachable_verb);
```

Comment:
Improve cachable coompliance by adding http header check on no-cache/no-store values.

```suggestion
    private function isCachable(string $method, EsiResponse response): bool
    {
        if (! in_array(strtolower($method), $this->cachable_verb))
            return false;

        if ($result->expired())
            return false;

        if (in_array('no-cache', $result->getHeader('Cache-Control')))
            return false;
            
        if (in_array('no-store', $result->getHeader('Cache-Control')))
            return false;

        return true;
```